src/reducers/index.js

```diff
@@ -12,9 +12,20 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {
+      ...state,
+      count: state.count+1
+    }
     case DECREMENT:
-    // Fill in the body of this case
+    return {
+      ...state,
+      count: state.count-1
+    }
+    case INCREMENTODD:
+    return {
+      ...state,
+      count: state.count % 2 !==0 ? state.count+1 : state.count
```

Comment:
beautiful logic with the ternary -- move this into you incrementIfOdd function in Count.js and you've removed the need for this third actioncreator/action/reducer flow!